version_bump.sh

```diff
@@ -8,13 +8,29 @@ function usage {
     echo "    ./version_bump.sh 2.6.0 2.6.1"
 }
 
-if [[ "$#" -lt 1 ]] ; then
-    usage
-else
+# Usage: updateVersion $arg1 $arg2
+function updateVersion {
+    echo
+    echo "Updating version from $1 to $2"
```

Comment:
If we `source core.sh` we would be able to just use `print_heading` https://github.com/Workiva/w-mobile-kit/blob/master/core.sh#L14, but we only have two prints right now so don't need to do so yet.